homeassistant/components/wled/coordinator.py

```diff
@@ -34,6 +37,15 @@ def __init__(
             update_interval=SCAN_INTERVAL,
         )
 
+    @property
+    def has_active_master_light(self) -> bool:
```

Comment:
This tightly couples the update coordinator to entities, creating a circular reference. Instead, can you extract this information from the config entry + WLED data directly?

Reply:
Simplified it a lot more, while the same test suite passes.